src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/Markup/DateTimeConverter2.cs

```diff
@@ -45,27 +40,21 @@ public override bool CanConvertTo(ITypeDescriptorContext context, Type destinati
             return base.CanConvertTo(context, destinationType);
         }
 
-
         public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
-        {
-            return _dateTimeValueSerializer.ConvertFromString( value as string, _valueSerializerContext );
-        }
-
-
+            => _dateTimeValueSerializer.ConvertFromString( value as string, null );
 
         public override object ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType)
         {
             if (destinationType != null && value is DateTime)
             {
-                _dateTimeValueSerializer.ConvertToString( value as string, _valueSerializerContext );
+                _dateTimeValueSerializer.ConvertToString(value as string, null);
```

Comment:
🚩 this doesn't return the actual value?!

Reply:
@hughbe My apologies for missing this while reviewing. It appears that this should return:
    ``` return _dateTimeValueSerializer.ConvertToString(value as string, null);```
    
Regarding the build failure, we cannot define ```StringHelpers``` as private. 

If you would like, I can add a commit with the above changes, as they are very small.